src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -116,8 +117,10 @@ class DocAnnotator extends Annotator {
         this.highlightCurrentSelection = this.highlightCurrentSelection.bind(this);
         this.createHighlightThread = this.createHighlightThread.bind(this);
         this.createPlainHighlight = this.createPlainHighlight.bind(this);
+        this.highlightCreateHandler = this.highlightCreateHandler.bind(this);
+        this.onTouchStart = this.onTouchStart.bind(this);
 
-        this.createHighlightDialog = new CreateHighlightDialog();
+        this.createHighlightDialog = new CreateHighlightDialog(undefined, this.isMobile);
```

Comment:
Yes! In this case, I didn't include a parent bc it incurs ONE unnecessary transfer from one parent to another, the first time it's shown (we move it from one pageEl to another as it's used).